experimental/ir/ir_file.go

```diff
@@ -231,10 +240,10 @@ func (f File) Options() seq.Indexer[Option] {
 	)
 }
 
-// Symbols returns this file's transitive symbol table.
+// Symbols returns this file's symbol table.
 func (f File) Symbols() seq.Indexer[Symbol] {
 	return seq.NewFixedSlice(
-		f.Context().symbols,
+		f.Context().imported,
```

Comment:
Shouldn't this be `exported`? The Go docs aren't clear as to what exactly it returns, but I'd expect just based on the name and brief doc that it would be the symbols defined in the file.

Reply:
Nope. Fixed the doc.